run.py

```diff
@@ -1,4 +1,4 @@
 from app import app
 
 if __name__ == '__main__':
```

Comment:
Can update requirement and readme(how to run it in dev/prd env?)

Reply:
OK. We should also include how to deploy